sphinx/devsetup/index.md

```diff
@@ -48,6 +48,18 @@ The following directories/files are part of our legacy data platform and are sti
 - `value_locked_flows`
 - `Pipfile`
 
+For the deployment of new pipelines, the general flow of work should be as follows:
+- Proof of Concept notebook
+- An update secrets with latest credentials
```

Comment:
add in `.env` file locally?

Reply:
I would remove secret managment here as we document internally how it should be done and it's only done if needed, not on every pipeline development